### ISSUE TITLE
GeoJSON doesn't load as expected (in BETA version)

### ISSUE THREAD

**Issue by programmeraditya** [googlemaps/android-maps-utils#810]
I have a GeoJSON file with a point and a hole. The point is basically the whole map and a specific area is the whole. 

GeoJSON file: https://pastebin.com/raw/krmeEeV0

The issue is that I get this type of rendering: 
![Screenshot_20210103-180459__01](https://user-images.githubusercontent.com/24489590/103478451-e2597200-4dee-11eb-9fad-5fcb1a148514.jpg)

It used to work fine on the stable Maps SDK. However with the BETA version of Maps SDK and Utils, it's loading like this.

**Comment by barbeau** [comment]
@programmeraditya I'm not sure the GeoJSON file that you linked to is valid. Here's what http://geojson.io/ renders:

![image](https://user-images.githubusercontent.com/928045/103569835-0328d280-4e96-11eb-8405-d4fc63c90d62.png)

...and https://geojsonlint.com/ says:

![image](https://user-images.githubusercontent.com/928045/103569888-176ccf80-4e96-11eb-8a4a-1966a9dc6e26.png)

Could you double-check the GeoJSON to make sure it's valid?

Some of the polygon drawing gets unpredictable on the Maps SDK if the GeoJSON file isn't valid.

**Comment by programmeraditya** [comment]
Thank you for linking to geojsonlint.com. This gives me a clearer idea that the GeoJSON's first set of coordinates didn't follow the right-hand rule. I have converted the first set to follow the right-hand rule.

For anyone else looking to fix the right-hand rule, they can easily convert the GeoJSON using this online page: https://mapster.me/right-hand-rule-geojson-fixer/

**Comment by programmeraditya** [comment]
@barbeau I may have responded too quickly. It looks like there is indeed something wrong in the Beta version of Maps SDK.

Corrected GeoJSON file: https://pastebin.com/raw/99hjSMGc

Even after correcting the file, the SDK loaded it incorrectly. This file works perfectly on both https://geojson.io and https://geojsonlint.com. Also I'm pretty sure the previous version of the file worked well with the stable Maps SDK.

![Screenshot_20210105-010353](https://user-images.githubusercontent.com/24489590/103570918-15385e80-4ef2-11eb-9fe5-91868be845e1.jpg)

You can see in the above screenshot that the hole was created but it was also colored.

**Comment by barbeau** [comment]
@programmeraditya Thanks for the fixed GeoJSON! If you're still seeing issues the problem is likely with the Maps SDK v3 beta and not this android-maps-utils library.

If you could confirm that this works as intended when using `implementation 'com.google.maps.android:android-maps-utils:2.2.0'` instead of `implementation 'com.google.maps.android:android-maps-utils-v3:2.2.0'`, please open an issue on the Maps SDK v3 beta at https://issuetracker.google.com/issues/new?component=541018&template=1201436.

**Comment by barbeau** [comment]
@programmeraditya I tested the GeoJSON at https://pastebin.com/raw/99hjSMGc with both the v2 and v3 beta libraries using the `gms` and `v3` product flavors respectively that I created in https://github.com/googlemaps/android-maps-utils/pull/811. I loaded the GeoJSON up in the `GeoJsonDemoActivity` of the demo app (by just dumping the GeoJSON into the earthquakes_with_usa.json file) and this is what I see on a Samsung Galaxy S8+ w/ Android 9:

![image](https://user-images.githubusercontent.com/928045/103826100-34df9c00-5044-11eb-94b7-756d072453e1.png)

So they both look the same to me. Is it possible that something else is going on that's impacting what you're seeing in your screenshots?
